src/utils/dom.js

```diff
@@ -15,6 +15,12 @@ export const states = {
  * Add modal + overlay to DOM
  */
 export const init = (params) => {
+  // Clean up old modals
+  while (getContainer()) {
```

Comment:
> If I remove the lines now the tests break

For me tests are passing with this code:

```js
const c = getContainer()
c && c.parentNode.removeChild(c)
```

or

```js
const c = getContainer()
if (c) {
  c.parentNode.removeChild(c)
}
```

Which is basically the same.